Workiva/w-mobile-kit
# MOBILE-2519: AutoViewLayoutVC

Description
---
New AutoViewLayoutVC that takes in an array of views and displays them in a wrapped view with an automatically generated height based on the content. 

What Was Changed
---
- Fixed smithy so it fails if the example app fails to build
- Implemented AutoViewLayoutVC
- Added to WUtils with randomly generated views and colors
- Unit tests for new functionality

Testing
---
- Verify changes in example app
- Verify uses of AutoViewLayoutVC
- Verify rotation and various sizes of AutoViewLayoutVC

---

Please Review: @Workiva/mobile
